### DISCUSSION TITLE
cuid2

### DISCUSSION THREAD

**Discussion by ethndotsh** [prisma/prisma#17549] [Q&A]
Saw that CUID is now not recommended for use because of security concerns. Are there plans to add a `cuid2`/ replace `cuid` in Prisma or are we going to keep using the old CUID.

**Comment by janpio** [comment]
Yes: https://github.com/prisma/prisma/issues/17102

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
